templates/CreateNativeTxtFiles.ps1

```diff
@@ -0,0 +1,60 @@
+﻿<#
+.SYNOPSIS
+Export all "export functions" from a native Win32 dll.
+.DESCRIPTION
+TODO
```

Comment:
I suspect it's better to drop the description altogether until we have something to fill it with.